.github/workflows/ci.yml

```diff
@@ -94,11 +94,10 @@ jobs:
     steps:
       - name: 'Check out repository'
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-      - name: 'Set up JDK 11'
+      - name: 'Set up JDK 21'
         uses: actions/setup-java@9704b39bf258b59bc04b50fa2dd55e9ed76b47a8 # v4.1.0
-
         with:
-          java-version: 11
+          java-version: 21
```

Comment:
I had been a little scared to see what would happen when using a newer JDK for _docs_. But it's looking like an improvement: https://github.com/google/guava/issues/6790#issuecomment-1988762401